templates/minimal/README.md

```diff
@@ -81,15 +95,18 @@ be used to run the minimal template's runtime. `polkadot-omni-node` binary crate
 
 Please see installation section on [crates.io/omni-node](https://crates.io/crates/polkadot-omni-node).
 
-#### Build `minimal-template-runtime`
+#### Build `minimal-template-runtime` (A reminder to  build on the most updated version of polkadot-sdk master)
 
 ```sh
 cargo build -p minimal-template-runtime --release
 ```
 
 #### Install `staging-chain-spec-builder`
+  ```bash
+  cargo install --path substrate/bin/utils/chain-spec-builder
+```
 
-Please see the installation section at [`crates.io/staging-chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder).
+Or Please see the installation section at [`crates.io/staging-chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder).
```

Comment:
Better to revert this because we will assume this readme lives in https://github.com/paritytech/polkadot-sdk-minimal-template.